### PR TITLE
[CBRD-24020] Supports error messages in DBLINK-related multilingual environment

### PR DESCRIPTION
**Pull request by ctshim**
http://jira.cubrid.org/browse/CBRD-24020

Purpose
Error messages for error codes added to support DBLINK  are reflected in multilingual files.

Implementation
N/A

Remarks
N/A

### CODE REVIEW

File: msg/ko_KR.utf8/cubrid.msg [hunk 1]
```diff
@@ -1355,7 +1355,24 @@ $ LOADDB
...
+1304 dblink - 암호화할 비밀번호의 길이가 최댓값을 넘어섰습니다.
+1305 dblink - 암호화된 비밀번호의 길이가 올바르지 않습니다.
+1306 dblink - 암호화된 비밀번호의 체크섬이 일치하지 않습니다.
+1307 dblink - 암호 문자열의 형식이 올바르지 않습니다.
+1308 dblink - 암호화된 비밀번호를 복호화하는 데 실패했습니다.. error=%1$d".
+1309 dblink - 비밀번호를 암호화하는 데 실패했습니다. error=%1$d".
+1310 dblink - 이름이 "%1$s"인 서버가 여러 개 있습니다.
+1311 dblink - 소유권을 갖는 "%1$s" 서버가 없습니다. 
+
+1300 마지막 에러
```
Comment (beyondykk9) [inline msg/ko_KR.utf8/cubrid.msg]: please, change to 1312.

File: msg/tr_TR.utf8/cubrid.msg [hunk 2]
```diff
@@ -1355,7 +1355,24 @@ Anahtar dosyasının (_keys) yolunu kontrol edin ve uygun anahtarı içerdiğind
...
+1304 dblink - password length exceeds max size.
+1305 dblink - encrypted password length is incorrect.
+1306 dblink - the checksum of the encrypted password does not match.
+1307 dblink - invalid cipher string format.
+1308 dblink - Failed to decryption password. error=%1$d".
+1309 dblink - Failed to encryption password. error=%1$d".
+1310 dblink - Multiple "%1$s" Server found.
+1311 dblink - Not allowed "%1$s" Server. 
+
+1300 Son Hata
```
Comment (beyondykk9) [inline msg/tr_TR.utf8/cubrid.msg]: please, change to 1312

File: msg/zh_CN.utf8/cubrid.msg [hunk 3]
```diff
@@ -1356,7 +1356,24 @@ $ LOADDB
...
+1304 dblink - password length exceeds max size.
+1305 dblink - encrypted password length is incorrect.
+1306 dblink - the checksum of the encrypted password does not match.
+1307 dblink - invalid cipher string format.
+1308 dblink - Failed to decryption password. error=%1$d".
+1309 dblink - Failed to encryption password. error=%1$d".
+1310 dblink - Multiple "%1$s" Server found.
+1311 dblink - Not allowed "%1$s" Server. 
+
+1300 最后一个错误.
```
Comment (beyondykk9) [inline msg/zh_CN.utf8/cubrid.msg]: please, change to 1312

File: msg/fr_FR.utf8/cubrid.msg [hunk 4]
```diff
@@ -1355,7 +1355,24 @@ Vérifiez le chemin du fichier de clé (_keys) et assurez-vous qu'il inclut la c
...
+1304 dblink - password length exceeds max size.
+1305 dblink - encrypted password length is incorrect.
+1306 dblink - the checksum of the encrypted password does not match.
+1307 dblink - invalid cipher string format.
+1308 dblink - Failed to decryption password. error=%1$d".
+1309 dblink - Failed to encryption password. error=%1$d".
+1310 dblink - Multiple "%1$s" Server found.
+1311 dblink - Not allowed "%1$s" Server. 
+
+1300 Dernière erreur
```
Comment (beyondykk9) [inline msg/fr_FR.utf8/cubrid.msg]: please, change to 1312

File: msg/ko_KR.utf8/cubrid.msg (hunk 1 above)
Comment (ctshim) [reply msg/ko_KR.utf8/cubrid.msg]: Thank you.

File: msg/zh_CN.utf8/cubrid.msg (hunk 3 above)
Comment (ctshim) [reply msg/zh_CN.utf8/cubrid.msg]: Thank you.

File: msg/fr_FR.utf8/cubrid.msg (hunk 4 above)
Comment (ctshim) [reply msg/fr_FR.utf8/cubrid.msg]: Thank you.

File: msg/tr_TR.utf8/cubrid.msg (hunk 2 above)
Comment (ctshim) [reply msg/tr_TR.utf8/cubrid.msg]: Thank you.

**Review by beyondykk9 — APPROVED** [review]

**Review by hgryoo — APPROVED** [review]

**Review by shparkcubrid — APPROVED** [review]

**Review by youngjinj — APPROVED** [review]

**Pull request merged**